tests/seeding/test_dataset_deletes.py

```diff
@@ -0,0 +1,255 @@
+            return FakeConditionTemplateCollection()
+
+        @property
+        def parameter_templates(self) -> ParameterTemplateCollection:
+            return FakeParameterTemplateCollection()
+
+        @property
+        def process_templates(self) -> ProcessTemplateCollection:
+            return FakeProcessTemplateCollection()
+
+        @property
+        def material_templates(self) -> MaterialTemplateCollection:
+            return FakeMaterialTemplateCollection()
+
+        @property
+        def measurement_templates(self) -> MeasurementTemplateCollection:
+            return FakeMeasurementTemplateCollection()
+
+        @property
+        def process_runs(self) -> ProcessRunCollection:
+            return FakeProcessRunCollection()
+
+        @property
+        def material_runs(self) -> MaterialRunCollection:
+            return FakeMaterialRunCollection()
+
+        @property
+        def measurement_runs(self) -> MeasurementRunCollection:
+            return FakeMeasurementRunCollection()
+
+        @property
+        def ingredient_runs(self) -> IngredientRunCollection:
+            return FakeIngredientRunCollection()
+
+        @property
+        def process_specs(self) -> ProcessSpecCollection:
+            return FakeProcessSpecCollection()
+
+        @property
+        def material_specs(self) -> MaterialSpecCollection:
+            return FakeMaterialSpecCollection()
+
+        @property
+        def measurement_specs(self) -> MeasurementSpecCollection:
+            return FakeMeasurementSpecCollection()
+
+        @property
+        def ingredient_specs(self) -> IngredientSpecCollection:
+            return FakeIngredientSpecCollection()
+
+    return FakeDataset()
+
+
+def test_wipe_dataset(fake_dataset):
+
+    failure_resp = {"failures": [
+        {
+            "id":{
+                "scope": "somescope",
+                "id": "abcd-1234"
+            },
+            "cause": {
+                "code": 400,
+                "message": "",
+                "validation_errors": [
+                    {
+                        "failure_message": "fail msg",
+                        "failure_id": "identifier.coreid.missing"
+                    }
+                ]
+            }
+        }
+    ]}
+    num_objs = len([
+        *fake_dataset.ingredient_runs.list_all(),
+        *fake_dataset.ingredient_specs.list_all(),
+        *fake_dataset.material_runs.list_all(),
+        *fake_dataset.material_specs.list_all(),
+        *fake_dataset.measurement_runs.list_all(),
+        *fake_dataset.measurement_specs.list_all(),
+        *fake_dataset.process_runs.list_all(),
+        *fake_dataset.process_specs.list_all(),
+    ])
+    num_templates = len([
+        *fake_dataset.material_templates.list_all(),
+        *fake_dataset.measurement_templates.list_all(),
+        *fake_dataset.process_templates.list_all(),
+        # uncomment these when attribute template delete is possible
```

Comment:
```suggestion
        # TODO: uncomment these when attribute template delete is possible
```
TODO makes it less likely to be overlooked